.github/workflows/common.yml.inactive

```diff
@@ -120,3 +122,14 @@ jobs:
     if: github.event_name == 'pull_request'
     with:
       r-version: $R_VERSION
+  cran-status:
+    name: CRAN Status
+    uses: pharmaverse/admiralci/.github/workflows/cran-status.yml@main
+    if: github.event_name == 'schedule'
+    with:
+      # Whom should the issue be assigned to if errors are encountered
+      # in the CRAN status checks?
+      issue-assignees: ""
```

Comment:
Can we place me, thomas, stefan, ross as the assigneess?  Just so we get a notification.

Reply:
We can, this will need to be done each individual repo. No consequences of setting it here.
If anyone outside of Pharmaverse uses this template and forget to change the default assignees, then you guys will be notified there as well 😄

Reply:
oh i see

Reply:
i say lets not set here adn do at individual repos